meta/recipes-devtools/python/python3-brotli_1.0.9.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Brotli compression format"
+SECTION = "devel/python"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/brotli-${PV}/LICENSE;md5=941ee9cd1609382f946352712a319b4b"
+
+# pypi package does not have a valid license file
+SRC_URI = "http://codeload.github.com/google/brotli/tar.gz/v${PV};downloadfilename=brotli-${PV}.tar.gz"
```

Comment:
This is not true - [Brotli-1.0.9.zip ](https://files.pythonhosted.org/packages/2a/18/70c32fe9357f3eea18598b23aa9ed29b1711c3001835f7cf99a9818985d0/Brotli-1.0.9.zip) does contain a LICENSE file

Reply:
Thanks for reminding, the commit is wrong